src/amm/uniswap_v3/mod.rs

```diff
@@ -883,7 +870,7 @@ impl UniswapV3Pool {
 
         if liquidity_delta != 0 {
             //if the tick is between the tick lower and tick upper, update the liquidity between the ticks
-            if self.tick > tick_lower && self.tick < tick_upper {
+            if self.tick >= tick_lower && self.tick < tick_upper {
```

Comment:
Is this correct? Can you link `v3-core` for rationale?

Reply:
Seconded.